src/components/organisms/pupil/OakLessonLayout/OakLessonLayout.tsx

```diff
@@ -1,22 +1,30 @@
 import React, { ReactNode } from "react";
-import styled from "styled-components";
+import styled, { css } from "styled-components";
 
 import { OakBox, OakFlex } from "@/components/atoms";
 import { parseSpacing } from "@/styles/helpers/parseSpacing";
 import { getBreakpoint } from "@/styles/utils/responsiveStyle";
 import { OakCombinedColorToken, oakDefaultTheme } from "@/styles";
+import { backgrounds } from "@/image-map";
 import { parseColor } from "@/styles/helpers/parseColor";
 
-type LessonSectionName =
-  | "overview"
-  | "intro"
-  | "starter-quiz"
-  | "video"
-  | "exit-quiz"
-  | "review";
+export const lessonSectionNames: string[] = [
+  "overview",
+  "intro",
+  "starter-quiz",
+  "video",
+  "exit-quiz",
+  "review",
+];
+
+export type LessonSectionName = (typeof lessonSectionNames)[number];
+
+type Phase = "primary" | "secondary";
 
 export type OakLessonLayoutProps = {
   lessonSectionName: LessonSectionName;
+  phase: Phase;
+  celebrate: boolean;
```

Comment:
Happy to call this something else other than `celebrate` if there are some suggestions. This is to toggle the svg on the quiz pages when the answer is correct.